browse/services/document/metadata.py

```diff
@@ -607,8 +623,10 @@ def get_session(app: object = None) -> AbsMetaSession:
     config = get_application_config(app)
     orignal_versions_path = config.get('DOCUMENT_ORIGNAL_VERSIONS_PATH', None)
     latest_versions_path = config.get('DOCUMENT_LATEST_VERSIONS_PATH', None)
+    show_email_secret = config.get('SHOW_EMAIL_SECRET', '')
```

Comment:
Is this normally determined by a per-user entry in the DB?

Reply:
It's an application-level variable.